[2024-03-29T08:22:19.140+0000] {local_task_job_runner.py:121} ERROR - Received SIGTERM. Terminating subprocesses
[2024-03-29T08:22:19.169+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 143
[2024-03-29T08:22:20.655+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
